README.md

```diff
@@ -243,6 +260,8 @@ Test your changes by running `npm test <package to test>` where `<package to tes
 
 This script uses [dtslint](https://github.com/microsoft/dtslint) to run the TypeScript compiler against your dts files.
 
+Once you have all all your changes ready, use `npm run test-all` to see how your changes affect other modules.
```

Comment:
It might be worth adding something like "If your package depends on packages that need an `npm install`, you can `npm run test-all` *first* to do that."

But I'm not sure the extra clutter is justified.

Reply:
I also figured this was good advice, but if you edit _any_ files then `test-all` doesn't work:

```
> definitely-typed@0.0.3 test-all
> node node_modules/@definitelytyped/dtslint-runner/dist/index.js --path .

Error: 'git diff' should be empty. Following files changed:
types/yaireo__tagify/index.d.ts
    at Object.getDefinitelyTyped (/Users/ortatherox/dev/typescript/DefinitelyTyped/node_modules/@definitelytyped/definitions-parser/dist/get-definitely-typed.js:22:15)
    at async Object.prepareAffectedPackages (/Users/ortatherox/dev/typescript/DefinitelyTyped/node_modules/@definitelytyped/dtslint-runner/dist/prepareAffectedPackages.js:15:16)
    at async Object.runDTSLint (/Users/ortatherox/dev/typescript/DefinitelyTyped/node_modules/@definitelytyped/dtslint-runner/dist/main.js:32:11)
```

We could treat this as a bug in test-all? If we do then we can make a dupe of this command called `test-changes` which is a lot more logical for someone fresh to the rpeo